data/brands/amenity/fuel.json

```diff
@@ -3050,14 +3050,22 @@
       }
     },
     {
-      "displayName": "NIS",
+      "displayName": "НИС Петрол",
       "id": "nis-a971aa",
       "locationSet": {"include": ["rs"]},
       "tags": {
```

Comment:
I found 300 element using this wikidata while some element have simple `brand=NIS` in taginfo:
![simpleNIS](https://user-images.githubusercontent.com/42690037/192101722-c384ae1c-c308-445a-b937-e70d56044349.png)
![300elements](https://user-images.githubusercontent.com/42690037/192101726-f4276872-85bd-4ff1-a5ba-80d288cd7702.png)

I suggest that add `"nis"` as a matchNames.